lib/adapters/mongodb/converters.js

```diff
@@ -6,7 +6,8 @@ const Hoek = require('hoek')
 const mongoose = require('mongoose')
 const uuid = require('node-uuid')
 
-module.exports = function() {
+module.exports = function(options) {
+    const baseUri = options.baseUri;
```

Comment:
I'd prefer if we stuck to the style we've used for previous repos, personally, but this is definitely we should discuss as a team!